src/message/sign.js

```diff
@@ -46,7 +49,7 @@ async function verifySignature (message) {
   const pubKey = await messagePublicKey(message)
 
   // verify the base message
-  return pubKey.verify(bytes, message.signature)
+  return await pubKey.verify(bytes, message.signature)
```

Comment:
This already returns a promise. This would add an extra promise to be resolved
```suggestion
  return pubKey.verify(bytes, message.signature)
```